poptorch_experimental_addons/cpp/replicatedalltoall.cpp

```diff
@@ -0,0 +1,79 @@
+#include "replicatedalltoall.hpp"
+#include <algorithm>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wsign-compare"
+#pragma GCC diagnostic ignored "-Wunused-parameter"
+#include <popart/graph.hpp>
+#include <popart/ir.hpp>
+#include <popart/op/collectives/collectives.hpp>
+#include <popart/opmanager.hpp>
+#include <popart/opserialiser.hpp>
+#include <popart/tensor.hpp>
+#pragma GCC diagnostic pop
+
+namespace popart {
+
+ReplicatedAllToAllOp::ReplicatedAllToAllOp(const OperatorIdentifier &_opid,
```

Comment:
This looks like a different clang-format to the config we've got checked in—were/are you able to run it?

Reply:
Just done so.